cluster-autoscaler/utils/drain/drain.go

```diff
@@ -38,6 +38,8 @@ const (
 	// PodSafeToEvictKey - annotation that ignores constraints to evict a pod like not being replicated, being on
 	// kube-system namespace or having a local storage.
 	PodSafeToEvictKey = "cluster-autoscaler.kubernetes.io/safe-to-evict"
+	// IgnoreLocalStorageVolumeKey - annotation that ignores (doesn't block on) a local storage volume during scale down
+	IgnoreLocalStorageVolumeKey = "cluster-autoscaler.kubernetes.io/ignore-local-storage-volume"
```

Comment:
Better naming suggestions are welcome.

Reply:
"safe-to-evict" is a widely recognized concept at this point, maybe we could reuse it, e.g. `cluster-autoscaler.kubernetes.io/local-volume-safe-to-evict`? I know we're not technically evicting the volume, but I think it conveys the intent better. Or maybe include "scale down" and "blocking" instead, like `cluster-autoscaler.kubernetes.io/scale-down-non-blocking-local-volume`? Just "ignore" doesn't tell much about what purpose it's being ignored for.

On a slightly related note, I don't think we should restrict ourselves to just one volume here, what if there's more that should be ignored? WDYT about making the value of the annotation a comma-separated list of volume names instead?